Userland/Applications/PixelPaint/Tools/Tool.h

```diff
@@ -95,6 +95,8 @@ class Tool {
     void set_primary_slider(GUI::ValueSlider* primary) { m_primary_slider = primary; }
     void set_secondary_slider(GUI::ValueSlider* secondary) { m_secondary_slider = secondary; }
 
+    static Gfx::IntPoint constrain_line_angle(Gfx::IntPoint start_pos, Gfx::IntPoint end_pos, float angle_increment = M_PI / 8);
```

Comment:
Description + commit message: the code states `M_PI / 8`, which is 22.5 degrees, not 15

Reply:
Commit message and description fixed. Thanks :^)